<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddVariableDialog</class>
 <widget class="QDialog" name="AddVariableDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Variable Name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="m_variableNameLineEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Variable Value Type</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="m_variableTypeCombo">
     <item>
      <property name="text">
       <string>double</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>float</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>sint32</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>uint32</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>sint16</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>uint16</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>sint8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>uint8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>bytearray8</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Variable Memory Type</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QComboBox" name="m_variableMemoryTypeCombo">
     <item>
      <property name="text">
       <string>RAM</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>EEPROM</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QTextEdit" name="m_variableDescriptionTextEdit"/>
   </item>
   <item row="8" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Variable Description</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Variable Memory Offset</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QSpinBox" name="m_variableOffsetSpinBox">
     <property name="maximum">
      <number>256</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddVariableDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddVariableDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
